Update lessons.yaml

ndporter opened the pull request:
Make the following types of changes throughout:
1. typo corrections
2. updates to lesson status (alpha, beta, stable)
3. corrections to lists of tools

kariljordan commented:
Awesome call Jonah! Lots of buzz on Twitter as well. We try not to post on our blog on Fridays because we don’t think Friday posts get the attention they deserve. Would you be fine with us posting this early next week?

jduckles commented:
No problem at all.

kariljordan commented:
@kariljordan will merge on 23-Sept (Monday)

serahrono commented:
thank you for the fantastic post @jduckles, and for reviewing and scheduling it for today, @kariljordan!

ndporter committed:
Update lessons.yaml

Make the following types of changes throughout:
1. typo corrections
2. updates to lesson status (alpha, beta, stable)
3. corrections to lists of tools

maneesha commented:
Thanks for going through this list @ndporter 

Could we also use this opportunity to clean up a couple of other things?
1. Lesson names are not consistent in the Carpentries mega-list and on each LP's website.  For example, in this list, we refer to DC's "Intro to R and RStudio for Genomics" but on the DC site we call the same lesson "Data Analysis and Visualization in R" 
1. In this yaml file, I recognize the key "tool" is a bit ambiguous.  I think the intent was to have that refer to the technology/language being taught and not the specific platform that uses that language.  Just as we only list "R" and not "R Studio" we should only list "SQL" and not "SQLite".  This part is something I noticed myself a week or two ago and was planning to open an issue about anyways.  

I know these points may be bigger changes - and I know you're also doing this in some of your final acts in your role - so I am happy to merge this first and return to the other points later.  Tagging @tobyhodges for review.

tobyhodges commented:
Thanks @ndporter for opening this, and @maneesha for finding the other inconsistencies. Regarding Maneesha's first point, I have opened https://github.com/datacarpentry/genomics-r-intro/pull/318 to adjust the lesson title there. If and when that is merged, we can adjust here and make everything consistent. On the second point, let's update the tool listings here before merging.

tobyhodges commented on data/lessons.yaml:
```diff
@@ -101,7 +101,7 @@
     name: Using Databases and SQL
-    description: An introduction to querying and managing relational databases with SQLite, for researchers and others who want to learn how to access, filter and summarise large vlumes of data.
+    description: An introduction to querying and managing relational databases with SQLite, for researchers and others who want to learn how to access, filter and summarise large volumes of data.
     tool: SQL; SQLite
```
```suggestion
    tool: SQL
```

tobyhodges commented on data/lessons.yaml:
```diff
@@ -399,15 +399,15 @@
     name: Data Management with SQL for Social Scientists
     description: An introduction to relational databases and the SQL language that can be used to query them, for social scientists who want to store and access data effectively. (alpha)
-    tool: SQL
+    tool: SQL; SQLite
```
```suggestion
    tool: SQL
```

tobyhodges reviewed (approved):
Happy for this to be merged after the listings of SQLite have been removed under `tool`

ndporter committed:
Remove SQLite tool listings

Co-authored-by: Toby Hodges <tbyhdgs@gmail.com>

ndporter committed:
Remove remaining SQLite tool listings

ndporter commented:
Updated to remove all 4 SQLite tool tags

maneesha commented:
Accidentally closed this without merging
